packages/components/src/DataViewer/RecordsViewer/Leaf/RecordsViewerLeaf.component.js

```diff
@@ -11,20 +12,57 @@ export function RecordsViewerLeaf({
 	className,
 	nodeHighlighted,
 	displayTypes,
+	measure,
 }) {
+	const ref = React.createRef();
+	const [isValueOverflown, setIsValueOverflown] = useState(false);
+	const [isLongValueToggled, setIsLongValueToggled] = useState(false);
```

Comment:
Can you change the name, because `toggled` doesn't indicate if it's opened or closed